src/feeds/views.py

```diff
@@ -1616,3 +1623,26 @@ def check_match_query(request, user):
                 else:
                     return False
     return True
+
+
+def extract_tags(feed):
+    tag_index = []
+    delimiter_string = string.punctuation.translate(str.maketrans({'#':'', '_':''})) + string.whitespace
+    feed_list = re.split('([' + delimiter_string + '])', feed)
+    feed_list = [i for i in feed_list if i != '']
+    for i in range(len(feed_list)):
+        if len(feed_list[i]) > 1 and feed_list[i][0] == '#' and feed_list[i][1] != '_' and not '#' in feed_list[i][1:] and not feed_list[i].translate(str.maketrans({'#':'', '_':''})).isdecimal():
```

Comment:
PEP 8 recommends:
 "and not '#' in " → "and '#' not in "
https://pep8-ja.readthedocs.io/ja/latest/index.html?highlight=not%20#id42

Reply:
I fixed it with https://github.com/s-tip/stip-sns/pull/95/commits/818f68dfffb7f65aff12ed99d7365ee5281861ac.